src/main/java/org/openlmis/requisition/domain/RequisitionStatus.java

```diff
@@ -19,6 +19,7 @@
 
 public enum RequisitionStatus {
   INITIATED(1),
+  REJECTED(1),
```

Comment:
No consequences. This is used internally in this class (db serializes to string). I just thought it would additionally mark that those two are pretty much the same in the requisition workflow.